kedro_neptune/hooks.py

```diff
@@ -35,6 +38,20 @@ def log_dataset_metadata(run: Run, base_namespace: str, name: str, dataset: Abst
         'version': dataset._version if hasattr(dataset, '_version') else None
     }
 
+    if isinstance(dataset, AbstractNeptuneDataSet):
+        file_to_upload = None
+        try:
+            file_to_upload = File.create_from(dataset.load())
+        except TypeError as e:
+            pass
+
+        if file_to_upload is None:
+            file_to_upload = File(
```

Comment:
We can additionally try to upload `as_html` besides raw data if `File.create_from` wasn't successful.

Reply:
It's ok to just upload as a file or as an Image/Matplotlib. Here we would like to provide a reproduction mechanism.